_data/sw_libraries.yml

```diff
@@ -60,6 +66,10 @@
           - sasl
           - server-time
           - userhost-in-names
+        SASL:
+          - ecdsa-nist256p-challenge
```

Comment:
Commit amended, based on the discussion on IRC it seems unlikely it is going to be added.